src/main/resources/scipooldev.properties

```diff
@@ -11,4 +11,5 @@ SESSION_TAG_CLAIMS=sub,userid,team,user_name,company,given_name,family_name
 # 3409013 scipooldev-external
 TEAM_TO_ROLE_ARN_MAP=[{"teamId":"3407239","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}, \
                       {"teamId":"3409012","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}, \
-                      {"teamId":"3409013","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}]
+                      {"teamId":"3409013","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}  \
+                      {"teamId":"3409011","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogExternalEndusers"}]
```

Comment:
This looks like a mistake -- isn't 3409011 the scipoolprod-external team? Shouldn't this be 3409013?